src/client/src/views/Scenes.js

```diff
@@ -9,6 +9,22 @@ module.exports = {
   view: function() {
     return m('.scenes', [
       m('h1', 'Scenes'),
+      m('select.language-input', //{
```

Comment:
Shouldn't the language input form be on another page?

It doesn't really make sense to change the language type when you already have been adding/editing scenes of a certain type.

If you look at Ebraheem's UI mockup, you will see the language input form is a screen that the user will see before you get to the scene editor.

This would allow the scene editor page to display attribute options that are specific to the language/format type.

So I would move this to a separate screen like in the UI mockup.